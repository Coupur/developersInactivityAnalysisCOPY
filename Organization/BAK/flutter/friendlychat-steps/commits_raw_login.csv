sha,author_id,date
31af3aa8dae1fa276b05386c55eb27f4009902b3,collinjackson,2016-05-05 18:16:56
c5ce38eaa15484193e35ab6eda1387d8f52b51fb,collinjackson,2016-05-05 19:59:51
7c2aca135010291ac7a3d0ebb9b205d5141ec1ae,collinjackson,2016-05-06 00:34:01
61c4c95a90edcad273a0621f65d2a4b1ca82f968,collinjackson,2016-05-06 17:37:11
2f3213d090d23c0f3318d290a099ec915db63a0d,collinjackson,2016-05-10 18:12:40
315dfcee9f27d3798b70908f9a64491891e7c5bf,collinjackson,2016-05-10 18:13:35
cd90c4925a4be263b749884c28fae9d2d4fe237c,collinjackson,2016-05-12 17:38:45
47a23e89692da809496c581eb2b31f7676d559d1,collinjackson,2016-05-16 21:26:28
d910d19103c8c57528fa130bfd969c026d8c6046,collinjackson,2016-05-17 20:57:32
9d76f7a5bd6f1a99889b261f2be12919b863c84b,collinjackson,2016-05-17 23:02:21
40d83905c36bbd09390cfe31fbc4b086dfc59c17,collinjackson,2016-05-17 23:26:42
80e0eea2e550e3c5befd8d078291bc0daccf1eac,collinjackson,2016-05-17 23:49:53
d35c997b6c2171d9fea67665e9051c10d3068180,collinjackson,2016-05-18 17:19:27
17a08f2373570059185284b1a2248719d3ce314d,collinjackson,2016-05-18 17:44:46
70f9a835785609160e037101626b64927b82220e,sethladd,2016-05-19 15:47:24
e365b0e9b3bd6bab199eec4afde10a09400f129e,sethladd,2016-05-19 15:48:31
ee71345365eaa2b56b9c6b332bba70d865b52cb6,sethladd,2016-05-19 16:46:16
5c219442873db53fb82b86dca57a07d3b7f65566,sethladd,2016-05-19 21:52:10
8d2ad44e435892a4db15b653f37c77901ef410ad,collinjackson,2017-02-22 19:17:57
e682542d498a99fe5462c5f356966b7f2fc44f3d,collinjackson,2017-04-03 21:31:49
3a7b59a531d5181e84d0d83d8a53824152bd83e9,collinjackson,2017-04-04 17:28:33
e345c7b66cd1d7e5a8022cd8e0d3a025f6607507,collinjackson,2017-04-04 17:29:14
83bdab3ae7f5383c288f2fc4f75fe60ba2b46d6a,collinjackson,2017-04-04 21:20:02
9fe690a5b160dd0b903eaaaa8ae9f885e1178163,collinjackson,2017-04-07 23:32:07
33664826a9a0b014846628963eca5711bcd27d94,collinjackson,2017-04-13 02:23:22
7c405a22c89a618076310b12a3ba1e5f3e03ee2c,collinjackson,2017-04-19 01:40:35
2cd7e39f6bdebdbd5525befb4d6a2b4e7de22159,collinjackson,2017-04-19 17:09:18
9b5e84f33c3fb54b96cb037c1df072068c331fec,collinjackson,2017-05-03 17:30:25
d0a96371cb9f18dfad3fc1d21dfca1211161160b,collinjackson,2017-05-15 17:52:21
189f0fce0df540d4478f8e37424faaada1763a65,collinjackson,2017-05-16 00:09:57
65da7f4429726c3956a630d050f642df2f79b0cc,collinjackson,2017-05-16 18:40:30
2336728cf67975081d7ddc5816c8cbca0db33383,collinjackson,2017-05-16 19:14:01
e5855b3e503449d4ebb1fa933d27070cd5eafd61,collinjackson,2017-05-17 17:44:10
292e31fd5171605d99f8bf9f4dcd69fae6b0e8e3,collinjackson,2017-05-19 22:30:37
d879b4fe6cf65eea2470744c9ea684849bbe48fe,goderbauer,2017-07-11 18:21:17
3c1f2728498070bbf65b75847d03e3014de8f71c,goderbauer,2017-07-11 18:21:56
7f751f841356aa217c8666a84b3a8695ef0cb738,goderbauer,2017-07-11 18:23:33
ba5b0dee834ad8dcae74eef5d0e1f3957b69a6d7,goderbauer,2017-07-11 21:04:52
0b25429add33f575d0d879e0c9860f3d847adb38,goderbauer,2017-07-12 00:52:12
632ac500c02e14a57023d1fc2f46a4695f317b8b,LarkAscending,2017-07-13 23:03:38
899de918d70997130ee44b2359e5d7ebbea6ef19,LarkAscending,2017-07-13 23:21:08
0ea5ba431d200ba05be37a33592d3e37236cc928,LarkAscending,2017-07-13 23:43:51
89edb464f7a63374277fce77fda40eec85ceab48,LarkAscending,2017-07-14 01:28:04
d6955a5e186e27a9c8e13f22bcc75b6ee19c48bb,collinjackson,2017-08-30 21:58:01
80c2b54ba51f026d803a1bc02a72482463195eb4,leoybkim,2017-10-16 08:36:29
cbb3b1cc491ed2ecab2be5e5537f60292b41b5e4,LarkAscending,2017-10-31 15:29:26
95f6efdec5a0164ddb415fbdaeed0c7b0f42eed2,LarkAscending,2017-10-31 15:30:08
74f39e48cf3135b6d361a59927e2bb485c6f9a27,LarkAscending,2017-10-31 15:30:19
3fd78c8241636f2ee6faab1f457b297f61acca26,LarkAscending,2017-10-31 15:32:17
e2870427d57a0812dddba03527259e267fb302c3,LarkAscending,2017-10-31 15:32:59
0f7c09a3feb3a6ceaf3ff19789b051648086c1e3,LarkAscending,2017-10-31 15:33:05
0a85f4ccc8c3872f3cc3306ee3f5b4f6a3072adb,LarkAscending,2017-10-31 15:33:13
708a668461fc518b9441f043e8dabe2bd2a14dd4,LarkAscending,2017-10-31 15:38:49
0d08ca150bf5ab69deff6ea814866315341cac39,LarkAscending,2017-11-02 23:39:22
dead14f38a5d53ba8c206b40be02e0bd54298bbb,LarkAscending,2017-11-02 23:43:33
a1697ae95d178d36355d03c2eb61b43fd83e1b8f,LarkAscending,2017-11-02 23:44:03
693ca5859fea3c9c3383543711a76af54d7eafe8,LarkAscending,2017-11-02 23:44:34
82af57e192e9ac4d51f43c803e3dc9a87f15c615,LarkAscending,2017-11-02 23:45:06
d9565fae0ee487052a4c3c9eea312c4eec3ce907,LarkAscending,2017-11-02 23:45:43
1b58ce21d534f741f2828d25776d6d8fb1c982a9,LarkAscending,2017-11-08 00:08:52
7e6fe6a3267eff10070d753445407d93a618215f,LarkAscending,2017-11-08 00:09:56
fae7ff8ff59f6bf238d55d2ad5c6164b16053204,LarkAscending,2017-11-08 00:10:23
10f6c8c55f5770058b7b4c07bde5b51cf0b2a7b5,LarkAscending,2017-11-08 00:10:58
0359945cf57ba823efea7010103d70e467b43808,LarkAscending,2017-11-08 00:11:28
8c331134179ab7d268e85547a3dc0e369d1b26b9,LarkAscending,2017-11-08 00:12:03
dac711d53999cc16b4d2e4347f8bd0356bf32459,LarkAscending,2017-11-08 00:13:01
c36f1b27654de1a7bf54f885b67d601dea371358,LarkAscending,2017-11-08 00:15:02
b84e784705b7b4f33e690c38c9ae4ade58943145,LarkAscending,2017-11-09 01:41:12
d459f6d29ade9a8d7d52fa0a316899b56c81db94,LarkAscending,2017-11-09 01:41:38
456e32312e991e6543ceb2b6531153401f0dda22,LarkAscending,2017-11-09 01:41:54
a42006460a5e5ff6c9112a140260f3b670f6327c,LarkAscending,2017-11-09 01:42:23
50aa3712f33bbdfa39dbe4618d1b86673c9312ed,LarkAscending,2017-11-09 01:42:39
6d7b0fd7f94ca378f885a1d9301b95805b665a39,LarkAscending,2017-11-09 01:42:55
4337ff27d2bedffcb00c87cb0d668403318c4df4,LarkAscending,2017-11-09 01:44:53
e40f69a12a38d98ce6dce117e56e617f2b6f67d6,LarkAscending,2017-11-09 01:45:19
b06ed7a671db1c701893a5eefd2a23ffeb5af37a,LarkAscending,2017-11-09 01:45:40
d919ab58123a2adda1021e640c434ab922efcd8d,LarkAscending,2017-11-09 01:46:23
9f5bfc69ff05cc562285dca8d16d2a9b5cc8e380,LarkAscending,2017-11-09 01:46:49
759446f7895be8e8ec73d6e426f935926591deca,LarkAscending,2017-11-09 01:47:09
2805564fccf55d886d103e97343df5d564c26c65,LarkAscending,2017-11-09 01:47:32
883025d275a985cf8aae7ff996e3e27373b8b574,LarkAscending,2017-11-09 01:49:21
d2f1aa818cbdf83b3772a160eac53fd4b575391a,LarkAscending,2017-11-20 23:34:23
ad8a906016219a9256a0b2fc8f83fc62a37c4c3d,leoybkim,2017-12-05 23:32:44
c9b245d43634150f23fe6eefdddf9638af62e602,LarkAscending,2017-12-07 19:51:57
36d6ef7b6311a668a3b4308dad38cb846a720418,saied89,2018-04-17 17:30:43
d26146df72e63d07bea447b44bf04db1a06c07c8,yagitatsu,2018-04-25 05:54:17
bb9e48e1c4c8d89ce113d34c829ec16b14fee77a,efortuna,2018-05-23 00:35:11
07f5c9c46d3d357266666a487fcc3109b6cdcf5a,sfshaza2,2018-05-29 22:58:10
177423f8ef86e43f78c8330682c8210970a75436,sebasbad,2018-09-07 12:56:00
deac42d713ebd38bf39149778aef78cd18f5f070,sfshaza2,2018-09-07 14:51:51
