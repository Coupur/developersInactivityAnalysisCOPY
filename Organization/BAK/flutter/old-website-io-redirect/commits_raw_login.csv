sha,author_id,date
9ba2fad4c4f7cf07c6dd564558b79cbe047f50a5,kf6gpe,2019-02-21 23:14:16
25e2de02cfab6afc4063c174b9a7a46650c70796,filiph,2019-02-22 02:47:22
ca9a431620e3a2b970678fcb627c002f05722d09,kf6gpe,2019-02-22 03:50:07
7a06d62784e1f7f830bcddd19cdc4abf2f2444c0,kf6gpe,2019-03-12 18:21:21
