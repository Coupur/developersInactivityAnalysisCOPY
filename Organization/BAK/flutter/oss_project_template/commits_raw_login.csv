sha,author_id,date
0e214e25a7e66d0bc8b8d39b6e9aea1b223fc667,sethladd,2015-10-20 18:52:12
cb741f227ad811613330f3e47f8e874e71d3f49a,devoncarew,2015-10-20 19:23:23
9fa6b5bb70d3d6c5d75aaa3cfac70c61ae573201,sethladd,2015-10-22 01:56:11
445824e874d8405410cb747cc11dcb70d858b868,chinmaygarde,2016-02-04 23:52:14
64ee21f4c130a4a6da48ddbb58a45176df276237,chinmaygarde,2016-02-04 23:54:14
0da41731ce0a61a109367651711feca75c871a2b,collinjackson,2016-02-18 17:44:52
04c1b8e5bc29ec09588c2830ac75dd424588c092,collinjackson,2016-02-18 18:35:58
5424671c37a3e01660233a8ea9424ea41cbfb642,kulakowski,2017-04-19 20:40:37
1de36d5a469f3a74bad0b400217f89e3f69b3445,sethladd,2017-09-13 17:33:48
67d5534f9014187ac01509998b666291c3b60d1a,sethladd,2017-09-13 21:12:43
