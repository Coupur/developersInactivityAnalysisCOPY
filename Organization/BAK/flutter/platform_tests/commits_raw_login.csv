sha,author_id,date
8f08147990ca8ec2ae8bf172ac137309b8f12647,xster,2019-02-28 01:37:40
3dda3358c9f835fff4934ac63446cf40ffe65b0d,xster,2019-02-28 01:53:01
41af9a696ec9221ecba618ab628ff314d13a121c,xster,2019-02-28 02:33:44
