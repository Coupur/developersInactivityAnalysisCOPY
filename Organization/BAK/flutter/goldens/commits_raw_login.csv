sha,author_id,date
0ea80e6a0147f1a3a59ff57f460a3f038a0d2748,tvolkert,2018-05-01 17:23:17
a3923c1a6ed192ad55f18bef6999422d11fd7b1b,Hixie,2018-05-07 22:44:31
b039f23a9d0617f6a092c8e0ee939afb25f4a3b2,xster,2018-05-08 00:35:29
e3f3b6766b18e2461c89a371be6e30045d8e404f,Hixie,2018-05-08 19:39:41
ed07132380cd2db6dd664be1479c5b89579e3304,xster,2018-05-10 20:24:03
df3f713cc47fc4e919d1c5cf52c0543424c26c09,jason-simmons,2018-05-11 17:55:49
7b308e37d33605bbb535df2b5d596a8697ac78bd,tvolkert,2018-05-14 17:19:56
298b585e6eb2bf51b12cd0ea9261a1b7a83c9a29,tvolkert,2018-05-14 21:27:08
8edb99b86648f49f58a2268260a4dccb20df4be1,Hixie,2018-05-31 05:49:11
95f60c7e328b53dd65de18a608830a872a92e3cd,Hixie,2018-06-01 05:07:04
a30f6886b4498484f1dfd799fd97a8ea6f162c18,xster,2018-06-01 19:42:33
3501bd12ebcf6e81cb18f6a52dac6bf5971aae71,HansMuller,2018-06-04 22:53:13
17b9d558228edad27729a0c1823430d73cb26608,HansMuller,2018-06-05 14:36:23
760320e3fc8ccd12deb4066bd4033a98d078359f,xster,2018-06-05 21:35:30
568342373b14fab81fd665d397c8c00db3d46fca,HansMuller,2018-06-08 22:33:28
828a92a6cbe9d5eaa4e23d81334232651f4d5159,xster,2018-06-14 20:00:55
eb2726862b939c8dd96b946d2c22a3f217c95f0d,Hixie,2018-06-21 18:24:17
4d6510a08b6c4d0cf4db2e6dcef2c7b4b76748f9,liyuqian,2018-06-26 00:04:21
aa202474cf62deea7afbaab0a57e97c9a5c950ba,liyuqian,2018-06-27 17:10:32
c0e7174ec1fad55e920b9dfd8b79291e093a9f2d,liyuqian,2018-06-28 19:25:19
76747f8ecb7d4d38f4560d4487cc81acf9f7752b,matthew-carroll,2018-07-03 00:55:08
bc99a4bdea0533994b34bc49f160af75f4904952,goderbauer,2018-07-03 21:28:30
413041fd6bbedd1c8bc6ca0b6ea6ea948079b669,xster,2018-07-03 22:59:19
703a480de2cc8ffde75b5eb3e26b1ce2068ab0fb,aam,2018-07-06 20:23:10
5688f0965d775e5ad7202984dae29b6c99985559,aam,2018-07-10 02:08:09
2694735bcfe755238f42da1be9a472e6795c9bdd,cbracken,2018-07-10 20:11:56
35f172d7c241a8112db53dd9ee3c29a13cf19eb8,cbracken,2018-07-11 18:36:43
f324a22a94bd6f103081949fc2665548008ca90d,doxuanviet1996,2018-07-12 23:11:22
85eb5e7b2ac4e750605a6ce61599179cf4dcc677,doxuanviet1996,2018-07-12 23:13:34
8e9ddf72770a22e0d8eb678b2086382d71ba954f,Hixie,2018-07-16 20:16:49
622e3bc8c7a0d36041d1f4bd005bce86e3e1c4fc,Hixie,2018-07-16 22:04:40
4948d5d7c0e07d092bd38511e5e159007425ec48,Hixie,2018-07-16 23:37:41
3baed8dd5820307627972d7a2c06287b3061bc64,amirh,2018-08-03 00:05:16
cb1fa8afd8f9bd7f48b9b52d30d145712976fabb,liyuqian,2018-08-03 23:56:24
64b7a3a7aef2fea9c7529d4834bf9eb3d85602d8,liyuqian,2018-08-04 20:36:44
b781c51e30f9eb5fd6975043643c48664d22bad0,nataliesampsell,2018-08-06 17:23:10
0e89d64ed5fe7da317762484259241778c2fe7fc,nataliesampsell,2018-08-06 17:24:16
c1f0d6a4fb6fc82069584d54a8dbd9d05005c869,liyuqian,2018-08-06 19:36:33
111e6e99ec35e4af012a2f4b12af95231ab38920,liyuqian,2018-08-06 20:19:22
46cf554baf4840c326bbceaa51b11534069bb557,liyuqian,2018-08-06 20:20:32
ad24e33211b4ebef6b338b8de2ee591f07fab155,liyuqian,2018-08-07 23:57:35
6c45fafdf5e3231406e1921285767f6ca8c05d6e,mkustermann,2018-08-15 12:06:11
d0800cf4170e77682173c19703f605c6795e0ff8,liyuqian,2018-08-17 19:52:27
54220baaf1ea0b314e0f3ebbd6afa08b46e4b256,liyuqian,2018-08-17 20:15:40
cc98e28c974eea0bd9a8e24591857ae6b5479795,liyuqian,2018-08-21 21:39:00
1a999092d10a22bc700214b257cd4890c5800078,liyuqian,2018-08-22 23:02:53
da615501c1032cb803b2a5623b07d7f4834d9640,jacob314,2018-08-29 20:59:05
157b8cebd10c8f8308540ded5dadae502febd48d,liyuqian,2018-09-04 19:33:51
a3c342fdca81de3a81e6b63c6de3460b0d4fa161,jacob314,2018-09-05 18:26:46
f86d91d87d3263613d48324b1a0ffb7e6a8526fe,GaryQian,2018-09-06 23:33:00
555a2f96848c31b8c0303555535d6b5e20686e6e,GaryQian,2018-09-06 23:33:50
5b0696a651738116b4d25a5b0a946b0951e45e6f,mklim,2018-09-07 17:15:51
2dec744d9e4962fa065e15eb34f87d8f3a315ffc,GaryQian,2018-09-07 20:55:49
47d63367fd7fccec8e862b6cce7cd5c10de98f1e,GaryQian,2018-09-07 20:56:34
080401b4daa283e66061af5505aa882dead04613,GaryQian,2018-09-07 22:06:39
2ea361e534bf39fd6b1a8abd9b04d5c01c3198e2,jonahwilliams,2018-09-10 21:50:06
130faab3aee8893fa165daecf4122cd0ea6bccf4,goderbauer,2018-09-11 09:39:01
d7089dc3cc314166616ba66c1978c9f8e9b9142e,GaryQian,2018-09-11 22:02:35
1d0b2f4175ca27957953bb4fb02f3113c3d3066e,goderbauer,2018-09-12 11:17:57
73aaafada280dbc726c6bfab510dd7b46fc7e3be,goderbauer,2018-09-12 11:41:47
da68caa0a261663858998e047c02560e94404e7a,GaryQian,2018-09-12 16:54:22
96ed11be7a52f9b8ace5af0a286d8d4415dbd78b,liyuqian,2018-09-13 18:17:19
a7f6061b8171f6fc82b6f437d13079ee26189438,alexmarkov,2018-09-17 22:21:56
26b603d6603f90fd636b8d74c9eec75466ff39a3,johnsonmh,2018-09-19 00:06:45
12765115d37519f55c30274456e1a7e7de5c6c8d,johnsonmh,2018-09-19 21:39:19
e2907079057dc0e507e5753470de20df7913d973,mraleph,2018-10-12 16:27:09
87fa3c457ee7826be74a8e9059e100fe2b7e9832,liyuqian,2018-10-12 18:01:23
fcb6a6cad08bd4a7d5c0157b9f83acadfa9be7e3,liyuqian,2018-10-12 18:13:36
bf46dc6b05154649926392e0fc042289e6172ca7,GaryQian,2018-10-15 20:16:27
2a4133039bc4f59613a73a6d871a7c0e6f2396fa,liyuqian,2018-10-17 23:43:51
afef1dc571f94a41751917d3946defba782335f7,GaryQian,2018-10-23 17:20:40
055a75c5a63bf2b2c5142b38b88829593908fd10,GaryQian,2018-10-24 23:41:01
ffe7e5ceaeccaa7c9419ec44eff96c31f4fc2601,beckler,2018-11-05 17:04:23
dd723e291a7997ce6b551b910447b64b3fe9ba7e,goderbauer,2018-11-05 22:24:13
8c478bbaf27447f3d612959705b305e7d1293526,beckler,2018-11-06 22:09:11
e07cc0cb4fdf912062e71a6fd97cc91478d6e3b9,liyuqian,2018-11-12 19:50:27
6cd63945213af370e1e9dbf2702166ca908f6cce,GaryQian,2018-11-16 01:08:30
baf0082301c4b6bfac79cde3708a6bc14e6cc27c,johnsonmh,2018-12-11 18:51:50
35e132218aeeeb528cd3f79bd7d108f021927ce6,xster,2018-12-19 01:47:23
c47f1308188dca65b3899228cac37f252ea8b411,xster,2018-12-19 02:09:14
f9486828ab6bb483297cd4daaba12ec86e549ac5,Hixie,2018-12-23 19:39:52
ef6e56f8d5bd3b32b184d200a25bb63a51f4a9e0,Salby,2019-01-09 21:56:00
034b2a540bc46375cf0c175a0fd512dcd46971e0,dnfield,2019-01-10 00:18:58
4c56796cd4a1f0d7bd197b1fbd198a03a3c8f3b5,Hixie,2019-01-14 04:56:15
7e3f945a906f9f1ffdcb1edba870f4533d2c2b86,Hixie,2019-01-14 04:58:24
02905560cf2c2132a7adea10943001f29325f3be,GaryQian,2019-01-15 23:14:54
ecbdcf8d56d3aba0212c2b1b923466023ef09b4e,goderbauer,2019-01-16 02:57:33
c926e350b321a0308f4d3f533825e0323aeedf52,liyuqian,2019-01-18 23:44:06
ec26eeebb44068599c100d3f0c2db7102b047737,rami-a,2019-01-22 18:48:40
021d7a08e7d2114949e82f8f7378116c082664e9,jslavitz,2019-01-22 23:43:03
cd4406f65813a10ccd0526071dc6368b676c108c,jslavitz,2019-01-23 00:45:45
24e9ab38cea4fd85f11ee5a3a491576037294bb9,dnfield,2019-01-23 00:55:57
6fc7ec65d51116c3f83acb5251e57e779af2ebbb,rami-a,2019-01-24 15:11:12
b530d67675a5aa9c5458b93019ce91e20ad88758,jslavitz,2019-01-29 01:51:00
801f9f06a654d9784d640bb7b8cfe02e59ea8f4c,johnsonmh,2019-01-30 00:14:09
fb91f19b24eeef61c4fc4f50ceed762ae2f78de7,rami-a,2019-01-30 15:52:50
75f8c51b45224d18baa81cfaf24a78e3695132e0,johnsonmh,2019-01-30 17:34:52
46a3d26acbb1b0d72b6b02c30f03b9dbda7d5bdf,rami-a,2019-01-30 17:52:39
b15e2eef0dfe05d1aaa5077edd7f431aa168e260,johnsonmh,2019-01-30 18:29:42
c5d6c6774d11ed1ef2e79b64ae949968fb660609,GaryQian,2019-01-30 23:36:59
418a3c74e88b50f4f3c552a50d03df4dd921aaa1,GaryQian,2019-01-31 00:37:37
cbd3fa445868962b7e910e498791755c988e9890,GaryQian,2019-01-31 00:48:04
fcb7bb1f2fd8b98acf224a967baa8f7f518dc733,johnsonmh,2019-02-05 01:42:47
503d5d3d90a8b88a45d924fa26650e40a4df72aa,Hixie,2019-02-05 04:52:03
ec90c64e598804d691c1c6bfcd191a63480e3053,Hixie,2019-02-05 04:59:55
62c8fddc9e32ee83d2e9b3465c794d4261b4f10d,GaryQian,2019-02-07 23:15:39
b42a8cd4be1be1d4650421609646bb3dae734993,johnsonmh,2019-02-08 23:05:11
e06279970c2ff4b11884b3a20c9670cfb4328aae,johnsonmh,2019-02-08 23:14:11
170f9cc332e8c02da8ec46253327d2693aa75edb,dnfield,2019-02-20 04:01:50
316817d949c1f7051e8a4898486ae7643ce31167,dnfield,2019-02-20 04:03:13
8587c2409ec1b674a7d451c8bcd8d37ed5175fdc,xster,2019-02-20 22:07:28
07881de0d5ddb4a84cb83b7d8965ff220ceac753,rami-a,2019-03-04 19:07:23
154ee905c58527c67fc4b0940e231d93d62a8a4e,johnsonmh,2019-03-04 20:37:08
f4720dae250b3a9cb2dfddf3d96dcc6da78f17db,GaryQian,2019-03-05 19:24:08
e27e697cd534e9595acd4459a0f38782c4f750fc,cbracken,2019-03-05 23:23:34
f7a81e05c4ab87d499986427051806e280a4b0c1,dnfield,2019-03-08 00:02:53
acdfdfe9a30abea1cc4ba27198f335ced5e6f38c,liyuqian,2019-03-09 00:12:42
30e9222088f4b13ce9624d820972577a55720c53,shihaohong,2019-03-11 22:14:13
f6c80cc09fed857f3f78995f60aef3c054de27f9,shihaohong,2019-03-11 22:15:29
0d9c593142e6036823e883aac5e1c3fc2f5ca708,shihaohong,2019-03-12 16:17:39
7e35e4ce3ebc454fb4b2b85449246006e79b387e,dnfield,2019-03-14 17:08:01
901ec95718ea0ba441a991590215865a838fdc5d,dnfield,2019-03-14 17:08:34
b0ae19c7cbc01ca4ec24675b9f5b19e8ff44bd76,liyuqian,2019-03-15 22:04:37
874c61ae53e835ec2057cb4fe6f4f44ca063011d,dnfield,2019-03-18 05:53:09
5a261a4720ec8efa35eb129fb8b5472f3f36597f,dnfield,2019-03-18 05:53:59
8d7603855f5c656615be9dfb3e1de40a1666fe01,dnfield,2019-03-19 18:12:03
894e9634cda6eb940d1ad4efb20b63a174bb7797,dnfield,2019-03-19 18:12:16
cb8264b1953000a603ecc2659ece3483859438a1,LongCatIsLooong,2019-04-10 18:44:06
3048ed023e5cc1d8e201f7abbba346e73cf5fa2c,GaryQian,2019-04-12 23:26:18
40a74e6c0487cc693b6f58abee89af266229b948,liyuqian,2019-04-16 20:33:07
678179c3367367b67320b93dbb5f593d97672f01,GaryQian,2019-04-17 01:18:29
78dfbee0485dbe335edb37899839ac32d219edc1,GaryQian,2019-04-17 20:03:47
651e319605df43286d73c5ac12d299573c2258c5,xster,2019-04-24 00:05:47
a9d3c3ebc6ee2921b1460f2fae460ef164d24bca,dnfield,2019-04-24 19:36:52
ce38c74f73c2e8d75f566d9a74dccf3d133413ed,dnfield,2019-04-24 19:37:19
09ebc5361187e9cc20ddc350dc047f95812c61a4,dnfield,2019-04-24 20:59:18
3232f5eaf45bd7e2d2bfa4f0c23381a6416b2a67,xster,2019-04-25 00:29:53
8057c8e1e0276a2ae7c26a0e04d54f339f3c51ca,xster,2019-04-25 00:32:42
fb5b76a38f35e6d6edf168d1d82b4a9c10d64eda,GaryQian,2019-04-26 22:44:14
48389eab93bc63bbccc31474e9aef098623e6dd1,GaryQian,2019-04-27 01:07:55
dadf2c9b8a8de8fc605e094a539410d83f700b34,liyuqian,2019-04-29 20:14:25
5e0cf0b9b347526d299718a4755e2d65cffacaba,GaryQian,2019-04-29 20:19:06
b84d2ee4688af77442e4c59c87c687e38f3c2711,GaryQian,2019-05-01 18:26:28
8ff56df10bac1edbfb3de13a1bc5f1700039ae94,justinmc,2019-05-21 23:23:17
25ab3c95b1ac02a41973c05f96e664370857ce55,justinmc,2019-05-22 16:07:21
d79941e3d9c6e761189ac8aefa3a40c7fb98413e,dkwingsmt,2019-05-29 23:34:01
fd5e241858352ccd40b2a80f8a546fb7d98158ce,GaryQian,2019-05-30 22:07:37
2eea0c710c4fca4c6655b4efc15b68d4b44cd404,LongCatIsLooong,2019-06-01 14:17:30
7e1c39c70da2aa15be37e827a4cd9b7df9ae5430,Piinks,2019-06-03 22:39:40
041efaf483a1cd011f4b4f6dcd04e0d5cad9436e,Piinks,2019-06-03 22:44:20
99210dbb3c857dd02bd12b324f128223b5d62505,Piinks,2019-06-03 22:52:27
7a929f1768afba0ed58348e038f74a356ccec4f4,GaryQian,2019-06-04 00:01:20
eb9c1d66709a1f8a0291076865fe387ceed96dca,dkwingsmt,2019-06-04 00:14:17
88858e381049a264e4350eb63ce8c8ea18f7286a,Piinks,2019-06-04 16:31:15
ba594dee85b2fb4dcb4121c1850a73493688eee5,Piinks,2019-06-04 18:04:22
c1edd8ee91194ddc4b7e28fd10ae4541b1a8ff22,Piinks,2019-06-04 19:26:50
